docs/user-stories.md

```diff
@@ -0,0 +1,111 @@
+# GLVD User Stories
+
+# USER-01
+
+As a user of Garden Linux I want to know about security issues in Garden Linux so I can operate my systems responsibly.
+
+## Acceptance Criteria 
+
+- [ ] The user can query for known CVEs of a list packages
```

Comment:
Should it not be the other way around? I think it's more important to query a release and get the CVEs back found within a given release? I might be just cherry-picking on the language side here.